File: Redes/AtvCRuby/servidor.rb
require 'socket'

def start_server(host, port)
  server_socket = TCPServer.new(host, port)
  
  puts "Servidor iniciado em #{host}:#{port}"
  
  client_socket = server_socket.accept
  data = client_socket.recv(1024)
  message = data.force_encoding('utf-8')
  
  puts message
  
  client_socket.close
end

HOST = 'localhost' 
PORT = 9000       

start_server(HOST, PORT)

File: Redes/CTransporte/tcp/servidor.py
import socket

def start_server(host: str, port: int):
        server_socket = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
        server_socket.bind((host, port))
        
        server_socket.listen(1)

        print(f'Servidor iniciado em {host}:{port}')

        client_socket, address = server_socket.accept()
        data = client_socket.recv(1024)
        message = data.decode('utf-8')

        print(message)

        client_socket.close()

if __name__=='__main__':
    HOST = 'localhost' #ENDEREÇO IP SO SERVIDOR
    PORT = 8000 #PORTA --

    start_server(HOST,PORT)





